Core/include/Acts/Seeding/SeedFinderConfig.hpp

```diff
@@ -51,7 +51,7 @@ struct SeedFinderConfig {
 
   // radial range for middle SP
   // variable range based on SP radius
-  bool useVariableMiddleSPRange = true;
+  bool useVariableMiddleSPRange = false;
```

Comment:
what is the reason for this change?

Reply:
This has been commented by @LuisFelipeCoelho above, I suppose.